trek.js

```diff
@@ -0,0 +1,53 @@
+let baseUrl = "https://trektravel.herokuapp.com/trips/";
+let clearPage = function () {
+  $('main').empty();
+  $('main').show();
+};
+
+$(document).ready(function () {
+  $('main').hide();
+
+  let tripListTemplate = _.template($('#trip-list-template').html());
+  let showTripTemplate = _.template($('#show-trip-template').html());
+
+  let tripsCallback = function (response) {
+    clearPage();
+    let generatedHtml = tripListTemplate({ trips: response });
+    $('main').append(generatedHtml);
+  };
+
+  let tripCallback = function (response) {
+    clearPage();
+    let reservationUrl = baseUrl + response.id + "/reserve";
+    let generatedHtml = showTripTemplate({
+      trip: response,
+      url: reservationUrl
+    });
+    $('main').append(generatedHtml);
+  };
+
+  let failCallback = function (response) {
+    $('main').prepend("<p id='callout alert'>AJAX Request Failed!</p>");
+  };
+
+  $('#get-trips').click(function() {
+    let url = baseUrl;
+    $.get(url, tripsCallback).fail(failCallback);
+  });
+
+  $('main').on('click', '#trip-list a', function () {
+    let url = baseUrl + $(this).attr('id');
+    $.get(url, tripCallback).fail(failCallback);
+  });
+
+  $('form').submit(function (event) {
```

Comment:
Because you're using underscore templates this form doesn't exist until you render the show trip template.  

After that trip is rendered on line 26, you can then add the event handler.